doc/tutorials/highlevel.md

```diff
@@ -37,7 +37,7 @@ default = 1.1
 ```
 
 In Elektra a specification is defined through the metadata of keys in the `spec` namespace. The specification above contains metadata for
-three keys the parent key (`@`), `@/mydouble` and `@/myfloatarray/#`. The `#` at the end of `myfloatarray/#` indicates that it is an array.
+three keys: 1. the parent key (`@`), 2. `@/mydouble` and 3. `@/myfloatarray/#`. The `#` at the end of `myfloatarray/#` indicates that it is an array.
```

Comment:
Maybe you can even make a proper markdown enumeration?

Please do not forget to add release notes and to make the build server happy.

Reply:
Thanks for the suggestion, I converted it into an enum and added release notes!